apps/website/vite.config.ts

```diff
@@ -148,8 +148,10 @@ const config: UserConfigVite & { test: UserConfigVitest } = {
JE938NnCPShh3OJ598Um98UhyDs00D56KLLvpPgL/55pvVJgGgDHLwDAIfzPdL0w+XOJh0orVLmcwIlzbk0Yceeqju4atn/T9siT0BLkL0Ab7ZZptl6IS2FlU53Y7m7bffrpADKntydgjMBOl4EW58fKgRDrjJeUyyaE428Mcee6xMmTJlAJZ+hXEjAee4/gsuuGDSgB922GHFgsgGoOjsAneOl6rYNXUXzRtvvLFuCyOHLLUCz8ur3LUW+EQvmtXCf/9IIeDRm2hur6evNVbf+eefXzbddNN26IQd+2X72WefjQjclhEw9oQVe0T4lVdeOcwGwG1hA1ybG9cDnIAICXB5fLKBezEW3YxXGOdYdY2DMVbfueeeO2nA/cL1K5nOFlLSCeiudSO8C9yi74dPgLPfvBrhxx577LCUkhwe4DPzw2fYbe+ciHD5GNTADXBt+hkdh40755xz6s/+jrgJOQU8Ed4FDrbaphT2SClXXHHFMP2A511KC93c3nHHHTcswgEHWyvCp/en/TDNY5x4fw44o0WMVmGgEuCJDND1eXvpPctkFMA///zzasNowNnDjti81FJLTQX85ptvHhF4jfDjjz9+VODeJ0zWLsW79ADngBrYYLoB+jg4xxxz1Cj3A0UOX2WVVSaDdznwwAOnivCkMza0EW5np2y88cal/1tmmD1+afod0UZ3Aqd3wgkn1JTijnGwjXBKrr766urwMIkTcAK4X42BDTADVccBzmFv6Sxou+yyS33RNQHqRxRx0EEHjQkcD/aA7RWtjyKyQLeceeaZ5auvvhoAB3sYcM61wEH3sonw/k//su6663ZlzvS5z3KjASecTSLahwfvOLxRnOxy8MEHV+DgKIluqUB0A+6llnf0o70T9+rijDPOqPMT4eSRpfZOPPHEmlISVfJ3crgJ3nv09+rFp66JLIBfdtllU0U44xZYYIH6cotj/wXo+HXIIYeMGuE+p3l9PBpoMn7++edy7bXXlg8++GDwtAb6AHj/i06f9f8fYW1gt69TRdrhhx9e726Mm9nWK1ure1KK1jdLbxF33XXXMUHbutoFbLDBBvXmzKwtmX/ooYfWRdO5G6+uv/769YPISKkj87Qff/xxue666+rnvfSDnUpWXTT7u5BhwJPDgY/S3AjvoeWttdZaKzJnuPVz3v4VaDfUe/FpAW2edy7ffvttmXPOOWte96HAUzGzxS9NuxRFRFtEx/MVaK+dX3vttTLXXHNNFSgtcGmp199nV+AucF70iO4u8NwIuZ4RHi/vWma0eAN3/fXX13zoA8RYqYNN3tbdfffdFXS2ZOwVDBy1oALv3faMliOOOKJ4v23dGO8DBNDeufiYHhu6wPW3wGuE9993D4AzNAsm4DXJ92+EklRjhU7kr7baatXJ8aKgCuj8+fLLL2tUMnK0ArTUY88uPyogt5VTitbHZk/JzjvvXJ+aemE6/si9K6644pgz/BL1o+3FF1+sGw1A6Q7sNnBiG4FYVuD9ffaQSSld4OAqot61ADfZsTdswO+7777jPn7RMV4LtCfAV54ffvih7hCkj0T0aMDZatyCCy5Y/4HIh1/paiKK18a+8vj3B36rgIYdvWAHuGsjAp8yZcoAOIMDPNvCABfhqsfZ3dLvGHCVcJEO/HgLzGgAgPYWUQT52AssPdmSmRfY9OVYv3N2cVx1bq7vjT5Qz2iq8VlPKgNakNHB7wCnWxHhvssCTndbXY8fvdNPP70C1+FugZocTriqpI+T6acYJMDbYutktzGtqYaMRx55pH73zFeVRBG7AARdoZ8z3TaAOa46Fzxss3/edttt68fqaQUPtMXQPzoFFjmOxwIe3S1wXFXcev1N+hAhqs5uhAd4G+HtneYQYIr5FDFItaj6hThaxJvnn3ZEtF0HXTXP9eW0wDmqKl3QOafX/ADXH9sSjWT430CL9GjgA1qODhdy85TxkezIZJO+pBRRHpvSuq6S0+v/DK3AnVAAuP+gSkqpg/qD2wgPcALbCCdUCSzXzbfiy6ci38Jm6+URtSAC7eYARq7aymBTC9w4cuNM2i5w5y1w4+jRggO8fzpaaaWVqg/vvPNOvfn+7SElTGIX6PriHx3x2Y1uU0p8MibjyOmdddZZNaUQpCaldIF3I9xkxnNKjVAGMCh9gEW2ORYxO44YpE20xDEyItd8gBLhcYDuVIAc05OFyzjBIyDoUDM+uo0x3pjMpzuV7tgPtuq8tZnu2BjgZLU6nBtTgfdfd1bgThjcRnj6CA1wkwOGwd0IDywQKSXT+Bju3LwYFOPJ5VDGRq55YIOumKsar7bn5gKYPBoZsYUutZ1nvkKP2oXtnM1sY4fz2Gyuc6WNcPJbPc6NI6MHeKBoGSkCE+GcMLhNKSaqBCUSKTVOTR+l5pObyqkWWIwnK3LJCKyRgBsb0OYr+ugAXKS5ToY6FnDz6AtsraJtK9gJhsjLXPPzZCWHYxA7M66yBJyjjNWKZDlcDSyKW+BRThDlhFOa6pyjAd7KIasFHuONJVdVIoPMbkpxvXUm8viQx9qYLvDoMjfzzVUCNz44d5x+bNimjxzVXOdK9Gr5khpdxpDR63+yGgLESYCLcMCdc4LSLnDXKARGjYEMiKMt8BhOXozQGsN4xxwi19jkX/IA98gqxgWSuY7TNxJwtgWQY2PbGll0mq/QGX9iN7tU18iLbudqUkqAZwxdxhpT5/c/yg6Ac5ij/p8vKSVKspgSYJzqmGCOEBjjyFBd5wQZWmO0+lO7wMk1zk2j0zHgqmOAUiNDq49suyCPt8IucmKj1lg6M0dLbgC3x+nT8kFgOiZHpTPjA5xu8uNXdBlnfu+8886rwDlKaDelREkb4XnEKUwEcZDDDApwRuXpCXDXzYvjMcx5wJJFBluMb/tdMz8yzEulS4SpCttiX44DQBsZYATcaC3ZGCmx2fzMBTo3m66M0arG4fs/PgYkTjpB+eIAAAAASUVORK5CYII="/></defs></svg>',
```

Comment:
Where did you get the image? I will go take a look and see if there's a true svg export. This is not pretty